config/applications.json

```diff
@@ -2838,5 +2838,21 @@
 		"description":"NDI, or Network Device Interface, is a video connectivity standard that enables multimedia systems to identify and communicate with one another over IP and to encode, transmit, and receive high-quality, low latency, frame-accurate video and audio, and exchange metadata in real-time.",
 		"link": "https://ndi.video/",
 		"winget": "NDI.NDITools"
+	},
+	"WPFInstallkicad": {
+		"category": "Pro Tools",
+		"choco": "na",
+		"content": "Kicad",
+		"description":"Kicad is an open-source EDA tool. It's a good starting point for those who want to do electrical design and is even used by professionals in the industry.",
+		"link": "https://www.kicad.org/",
+		"winget": "KiCad.KiCad"
+	},
```

Comment:
It is a bit bugged as I added it that PR but it also joined this one.

Reply:
@Real-MullaC 
If you look into your git logs (by running `git log` in the terminal), you'll notice that you have a commit which adds kicad app entry, this's because you were _At_ kicad branch _when_ you branched off to make a new branch, which's the branch you're using in this PR.

I hope I made myself clear, that branching off another feature/patch branch is.. it isn't recommended for a reason, which's running into issues like these (having commits from previous branches).

Reply:
@Real-MullaC I've updated my previous comment, as you won't be able to push to your branch because you're re-writing git history, therefore you need the `--force-with-lease` or `--force` flag.